torch/_inductor/codegen/cpp_wrapper_cpu.py

```diff
@@ -2311,14 +2352,28 @@ def val_to_arg_str(self, val) -> str:
             if config.abi_compatible:
                 assert len(val) > 0, "Empty array is not supported in C"
                 static = self.is_statically_known_list_of_ints(val)
+                type_hint = "bool" if isinstance(val[0], bool) else "int64_t"
+                if (
+                    type_ is not None
+                    and isinstance(type_, torch._C.ListType)
+                    and isinstance(type_.getElementType(), torch._C.OptionalType)
+                    and isinstance(
+                        type_.getElementType().getElementType(), torch._C.TensorType
+                    )
+                ):
+                    type_hint = "tensor"
+                    tmp_arg_list = ""
+                    for x in val:
+                        tmp_arg_list += f"&{x}_handle, "
+                    result = f"{{{tmp_arg_list}}}"
                 # Need to pass the array length because we can't use std::vector
-                int_var_array = self.codegen_int_array_var(
+                var_array = self.codegen_var_array(
```

Comment:
Basically this is the only place I expect to call `codegen_var_array` instead of `codegen_int_array_var`.

Reply:
I see. fixed in https://github.com/pytorch/pytorch/pull/126412/commits/89d69e2737992313054bd015771d1c2c20e784cb